open_event/templates/gentelella/admin/event/roles/details.html

```diff
@@ -0,0 +1,140 @@
+                                        Draft <br>
+                                    </span>
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="#step-2" class="disabled" isdone="0" rel="2">
+                                            <span class="step_no">2</span>
+                                    <span class="step_descr">
+                                        Call for papers<br>
+                                    </span>
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="#step-3" class="disabled" isdone="0" rel="3">
+                                            <span class="step_no">3</span>
+                                    <span class="step_descr">
+                                        Scheduling<br>
+                                    </span>
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="#step-4" class="disabled" isdone="0" rel="4">
+                                            <span class="step_no">4</span>
+                                    <span class="step_descr">
+                                        Published<br>
+                                    </span>
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="#step-5" class="disabled" isdone="0" rel="5">
+                                            <span class="step_no">5</span>
+                                    <span class="step_descr">
+                                        Completed<br>
+                                    </span>
+                                        </a>
+                                    </li>
+                                </ul>
+
+                                <div class="stepContainer" style="overflow-y: hidden;">
+                                    {% include '/gentelella/admin/event/details/steps/draft.html' %}
+                                    {% include '/gentelella/admin/event/details/steps/call_for_papers.html' %}
+                                    {% include '/gentelella/admin/event/details/steps/published.html' %}
+                                    {% include '/gentelella/admin/event/details/steps/completed.html' %}
+                                </div>
+                            </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+-->
+
+{% endblock %}
+
+{% block tail_js %}
+    {{ super() }}
+    <script src="{{ url_for('static', filename='admin/lib/interact/dist/interact.min.js') }}"></script>
```

Comment:
A lot of javascript assets here are not required to be included in this file anymore, since the scheduler has been moved.